src/tir/schedule/primitive.h

```diff
@@ -32,8 +34,8 @@ namespace tir {
  * \param max_exclusive The maximum value of the range, exclusive.
  * \return The random integer sampled in the given range.
  */
-TVM_DLL int SampleInt(support::LinearCongruentialEngine::TRandState* rand_state, int min_inclusive,
-                      int max_exclusive);
+TVM_DLL int32_t SampleInt(support::LinearCongruentialEngine::TRandState* rand_state,
+                          int32_t min_inclusive, int32_t max_exclusive);
 /*!
  * \brief Sample once category from candidates according to the probability weights.
  * \param self The schedule to update
```

Comment:
Could you please remove this line for me, thanks.

Reply:
We don’t need to remove this line. This API is good for internal use

Reply:
I mean the `self` param here is not shown in the function call so maybe we can remove it?

Reply:
oh sure, thanks for spotting this!